### PR TITLE
Update latex.rst

### PR DESCRIPTION
**Pull request by miketheman**
Subject: typo

### CODE REVIEW

File: doc/latex.rst [hunk 1]
```diff
@@ -111,7 +111,7 @@ If non-empty, it will be passed as argument to the ``\sphinxsetup`` command::
      ``\sphinxsetup``.
 
    - As an alternative to the ``'sphinxsetup'`` key, it is possibly
```
Comment (jfbu) [inline doc/latex.rst]: another typo: "possible" not "possibly" !

File: doc/latex.rst [hunk 2]
```diff
@@ -111,7 +111,7 @@ If non-empty, it will be passed as argument to the ``\sphinxsetup`` command::
...
-     to insert explicitely the ``\\sphinxsetup{key=value,..}`` inside the
+     to insert explicitly the ``\\sphinxsetup{key=value,..}`` inside the
```
Comment (jfbu) [inline doc/latex.rst]: on second thoughts I wonder if it would not be even better without any "explicitly" ?

**Review by jfbu — APPROVED** [review]
Thanks for catching this ! do you mind fixing also the "possibly" ?

**Comment by miketheman** [comment]
@jfbu Updated!

**Comment by tk0miya** [comment]
+1
@jfbu please merge this if you like it.

**Comment by jfbu** [comment]
Thanks !

**Comment by jfbu** [comment]
@miketheman I merged it also in stable for 1.5.4 release, thanks!

**Pull request merged**